src/Utility/Updater.lua

```diff
@@ -0,0 +1,52 @@
+--!nonstrict
+
+--[[
+    Centralizes all RunService connections so that the various parts of Fusion 
+    don't connect to RunService independently of one another.
+
+    You can optionally pause all internal updating by calling `:pause()`, and
+    then resume updating by calling `:resume()`.
+
+    If you want to manually control the update process, you can call
+    `.thread:Disconnect()`, since `.thread` is a Roblox Signal. After doing this, 
+    you then just need to account for throttling using the `.throttling` property,
+    and call `.stepped:Fire(dt)` whenever you want; `dt` here is just the change in 
+    time since the last update call.
+]]
+
+local RunService = game:GetService("RunService")
+
+local Package = script.Parent.Parent
+local Signal = require(Package.Parent.Signal)
+
+local class = {}
+class.__index = class
+
+function class.new()
+    local self = setmetatable({}, class)
+
+    self.stepped = Signal.new()
+    self.throttling = false
```

Comment:
The capitalisation is wrong here. `Throttling` is used in the 3 other cases, `throttling` is used here. In general, private members should be prefixed with _ and use camelCase. So that would mean `self._throttling = false`. Is throttling a great name, though? It's not very accurate to what it does. Maybe just do `self._paused`

Reply:
I forgot to push the version that fixed this when I read through it, thanks for the catch! 

With that being said, I would agree that ``self._paused`` is a better name. I was debating whether it should be private or public though, since if you decide to call ``self.thread:Disconnect()`` so that you can call ``self.stepped:Fire(dt)`` manually, you should take into account whether or not it is paused.

Since @Elttob wanted this to be internal, for now I will leave it as ``self._paused`` and for now not publicly expose the fact that you can call ``self.thread:Disconnect()``. To that end it might also be a good idea to make ``self.thread`` private aswell.

Reply:
Formatting now matches the Scheduler file. The issues you presented have been fixed.